contrib/gdk-pixbuf/loader.c

```diff
@@ -363,7 +363,7 @@ avif_image_saver (FILE *f,
         gchar **viter = values;
 
         while (*kiter) {
-            if (strcmp (*kiter, "quality") == 0) {
+            if (g_strcmp0 (*kiter, "quality") == 0) {
```

Comment:
Carlo: Thanks for the patch.

@novomesk Daniel: Could you please review this change? I don't know why `strcmp()` can't be used in this file (after we add `#include <string.h>`). Thanks!

Reply:
This is how the plug-ins in gdk-pixbuf repo looks like:

https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.43.2/gdk-pixbuf/io-jpeg.c#L1385

They use `strcmp` at similar portion of code, while using `g_strcmp0` in the same file too.

I would prefer to try  adding `#include <string.h>` after `#include <stdlib.h>` (like it is in GNOME/gdk-pixbuf repo) first.

Reply:
> Carlo: Please add `#include <string.h>`. Thanks!

Done!